docker-compose.yml

```diff
@@ -10,8 +10,16 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
   app:
-    image: opus10/circleci-public-django-app
-    environment:
-      - DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
+    build:
+      context: .
+      dockerfile: ./Dockerfile.dev
+    container_name: app
     depends_on:
       - db
+    stdin_open: true
+    tty: true
+    environment:
+      - DJANGO_SETTINGS_MODULE=settings
+      - ENVIRONMENT=development
```

Comment:
to fix the issue with pg connectivity and still use dj_database_url you need to keep the `DATABASE_URL=postgres://postgres:postgres@db:5432/postgres` here.

Reply:
Thanks, that worked :)